paper-pnas.tex

```diff
@@ -0,0 +1,219 @@
+\documentclass[9pt,twocolumn,twoside,lineno]{pnas-new}
+% Use the lineno option to display guide line numbers if required.
+% Note that the use of elements such as single-column equations
+% may affect the guide line number alignment. 
+
+\templatetype{pnasresearcharticle} % Choose template 
+% {pnasresearcharticle} = Template for a two-column research article
+% {pnasmathematics} = Template for a one-column mathematics article
+% {pnasinvited} = Template for a PNAS invited submission
+
+\title{How and Why to run a Hack Week}
+
+% Use letters for affiliations, numbers to show equal authorship (if applicable) and to indicate the corresponding author
+\author[a,b,1]{Daniela Huppenkothen}
+\author[c,1,2]{Anthony Arendt} 
+\author[b,d]{David W. Hogg}
+\author[e]{Karthik Ram}
+\author[f]{Jake VanderPlas}
+\author[g]{Ariel Rokem}
+
+\affil[a]{Center for Data Science, New York University, 65 5th Avenue, 7th Floor, New York, NY 10003, USA}
+\affil[b]{Center for Cosmology and Particle Physics, New York University, 726 Broadway, 10th Floor, New York, NY 10003, USA}
+\affil[c]{}
+\affil[d]{Center for Computational Astrophysics, Flatiron Institute, 162 5th Ave, New York, NY 10010, USA}
+\affil[e]{}
+\affil[f]{}
+\affil[g]{}
```

Comment:
The University of Washington eScience Institute, The WRF Data Science Studio, Physics/Astronomy Tower, 6th Floor, 3910 15th Ave NE,  Campus Box 351570
University of Washington, Seattle, WA 98105, USA